shared/wallets/common/participants-row/index.js

```diff
@@ -14,32 +14,41 @@ type Props = {|
 |}
 
 // A row for use in Participants components; provides a blue heading to the left of the content.
-const ParticipantsRow = (props: Props) => (
-  <React.Fragment>
-    <Kb.Box2 direction="horizontal" fullWidth={true} style={Styles.collapseStyles([styles.row, props.style])}>
-      <Kb.Text
-        type="BodyTinySemibold"
-        style={Styles.collapseStyles([
-          styles.headingText,
-          props.headingAlignment === 'Right' && {textAlign: 'right', width: 40},
-          props.headingStyle,
-        ])}
-      >
-        {props.heading}:
-      </Kb.Text>
-      <Kb.Box style={styles.childContainer}>
-        <Kb.Box style={styles.childFillContainer}>{props.children}</Kb.Box>
-      </Kb.Box>
-    </Kb.Box2>
-    {props.bottomDivider && (
-      <Kb.Divider style={props.dividerColor ? {backgroundColor: props.dividerColor} : {}} />
-    )}
-  </React.Fragment>
-)
+class ParticipantsRow extends React.Component<Props> {
```

Comment:
Might as well use PureComponent?

Reply:
ok but likely doesn't help as its driven by children which change all the time